src/superphot_plus/surveys/surveys.py

```diff
@@ -95,3 +95,17 @@ def ZTF(cls) -> Self:  # pylint: disable=invalid-name
         package_filepath = os.path.dirname(superphot_plus.__file__)
         yaml_file = os.path.join(package_filepath, "surveys", "ztf.yaml")
         return cls.from_file(yaml_file)
+    
+    
+    @classmethod
```

Comment:
Can you add a simple test to confirm that this loads, and has six bands, similar to `tests/superphot_plus/surveys/test_surveys.py:test_ztf`?

Reply:
yes!